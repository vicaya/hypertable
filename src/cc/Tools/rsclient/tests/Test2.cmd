UPDATE 'test/Test2' "Test2-data.txt";
CREATE SCANNER ON 'test/Test2'[..??] WHERE ROW >= "Abama" LIMIT=4;
DESTROY SCANNER;
CREATE SCANNER ON 'test/Test2'[..??] WHERE ROW >= "Abama" LIMIT=2 REVS=1;
DESTROY SCANNER;
quit
