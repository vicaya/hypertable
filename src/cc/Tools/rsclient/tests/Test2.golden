
Welcome to the rsclient command interpreter.
For information about Hypertable, visit http://www.hypertable.org/

Type 'help' for a list of commands, or 'help shell' for a
list of shell meta commands.

UPDATE 'test/Test2' "Test2-data.txt";
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
CREATE SCANNER ON 'test/Test2'[..??] WHERE ROW >= "Abama" LIMIT=4;
1189631333141746 Abama apple:http://haynese.winthrop.edu/india/medals/VC/1CSingh.html
1189631333575431 Abama apple:http://www.baseball-reference.com/r/roberda07.shtml
1189631333438063 Abama apple:http://www.usatoday.com/sports/bbw/columnist/nightengale/2005-01-19-majors_x.htm
1189631332964935 Abama apple:http://www.usatoday.com/sports/bbw/columnist/nightengale/2005-01-19-majors_x.htm
1189631333192949 Abama banana:http://www.baseball-reference.com/r/roberda07.shtml
1189631333239488 Abama banana:http://www.fcc.gov/fcc-bin/tvq?call=KOCB
1189631333289762 Abama banana:http://www.psa.ac.uk/awards2003/toynbee.htm
1189631333632042 Abama onion:http://sports.espn.go.com/mlb/players/profile?statsId=6305
1189631332983672 Abama onion:http://www.guardian.co.uk/Columnists/Archive/0,5673,-25,0.html
1189631333634436 Abama onion:http://www.housing.berkeley.edu/housing/
1189631333105993 Abama onion:http://www.mtv.com/news/articles/1508677/20050831/duff_hilary.jhtml?headlines=true
1189631333331091 Abama onion:http://www.nielsenmedia.com/DMAs.html
1189631333416064 Abama onion:http://www.usatoday.com/sports/bbw/columnist/nightengale/2005-01-19-majors_x.htm
1189631333479027 Abama cassis:http://haynese.winthrop.edu/india/medals/VC/1CSingh.html
1189631333309782 Abama cassis:http://pbskids.org/sesame/index.html
1189631333225486 Abama cassis:http://sports.espn.go.com/mlb/players/profile?statsId=6305
1189631333314019 Abama cassis:http://www.eighttribes.org/modoc/
1189631333487606 Abama cassis:http://www.klamathtribes.org
1189631333053367 Abama cassis:http://www.usdoj.gov/usao/iln/osc/documents/ag_letter_december_30_2003.pdf
1189631333511132 Abama cherry:http://toughpigs.com/anthernie00.htm
1189631333180300 Abama cherry:http://www.guardian.co.uk/Columnists/Archive/0,5673,-25,0.html
1189631333626253 Abama cherry:http://www.klamathtribes.org
1189631333049254 Abama cherry:http://www.lsbu.ac.uk/news-php/news.php?newsid=86
1189631333362748 Abama cherry:http://www.sesameworkshop.org/sesamestreet/
1189631333682271 Abama yam:http://haynese.winthrop.edu/india/medals/VC/1CSingh.html
1189631333379272 Abama yam:http://pbskids.org/sesame/index.html
1189631333064725 Abama yam:http://toughpigs.com/anthernie00.htm
1189631333023283 Abama yam:http://toughpigs.com/anthernie00.htm
1189631333228657 Abama yam:http://www.klamathtribes.org
1189631333659574 Abama yam:http://www.mtv.com/news/articles/1508677/20050831/duff_hilary.jhtml?headlines=true
1189631333397374 Abama yam:http://www.psa.ac.uk/awards2003/toynbee.htm
1189631333075652 Abama yam:http://www.sesameworkshop.org/sesamestreet/
1189631333036521 Abama yam:http://www.sesameworkshop.org/sesamestreet/
1189631333175254 Abama yam:http://www.sirlinksalot.net/bertandernie.html
1189631333674696 Abama plum:http://pbskids.org/sesame/index.html
1189631333353299 Abama plum:http://www.baseball-reference.com/r/roberda07.shtml
1189631333025117 Abama broccoli:http://www.sirlinksalot.net/bertandernie.html
1189631333633440 Abama broccoli:http://www.usdoj.gov/usao/iln/osc/documents/ag_letter_december_30_2003.pdf
1189631333089700 Abama broccoli:http://www.usdoj.gov/usao/iln/osc/documents/ag_letter_december_30_2003.pdf
1189631333707493 Abanic banana:http://haynese.winthrop.edu/india/medals/VC/1CSingh.html
1189631333605956 Abanic cassis:http://www.housing.berkeley.edu/housing/
1189631333670766 Abanic cherry:http://www.guardian.co.uk/Columnists/Archive/0,5673,-25,0.html
1189631333532516 Abanic cherry:http://www.housing.berkeley.edu/housing/
1189631333678356 Abanic yam:http://sports.espn.go.com/mlb/players/profile?statsId=6305
1189631333561403 Abanic yam:http://www.mtv.com/news/articles/1508677/20050831/duff_hilary.jhtml?headlines=true
1189631332824771 Abantes apple:http://www.housing.berkeley.edu/housing/
1189631333315720 Abantes apple:http://www.klamathtribes.org
1189631333507472 Abantes apple:http://www.lsbu.ac.uk/news-php/news.php?newsid=86
1189631333118353 Abantes apple:http://www.mtv.com/news/articles/1508677/20050831/duff_hilary.jhtml?headlines=true
1189631332878250 Abantes apple:http://www.mtv.com/news/articles/1508677/20050831/duff_hilary.jhtml?headlines=true
1189631333531952 Abantes apple:http://www.psa.ac.uk/awards2003/toynbee.htm
1189631332930705 Abantes apple:http://www.usatoday.com/sports/bbw/columnist/nightengale/2005-01-19-majors_x.htm
1189631332860729 Abantes banana:http://sports.espn.go.com/mlb/players/profile?statsId=6305
1189631333591185 Abantes banana:http://www.baseball-reference.com/r/roberda07.shtml
1189631333501763 Abantes banana:http://www.klamathtribes.org
1189631333266353 Abantes banana:http://www.klamathtribes.org
1189631332796971 Abantes banana:http://www.lsbu.ac.uk/news-php/news.php?newsid=86
1189631333090582 Abantes banana:http://www.mtv.com/news/articles/1508677/20050831/duff_hilary.jhtml?headlines=true
1189631332932608 Abantes banana:http://www.usatoday.com/sports/bbw/columnist/nightengale/2005-01-19-majors_x.htm
1189631333682700 Abantes banana:http://www.usdoj.gov/usao/iln/osc/documents/ag_letter_december_30_2003.pdf
1189631333128630 Abantes banana:http://www.usdoj.gov/usao/iln/osc/documents/ag_letter_december_30_2003.pdf
1189631332899000 Abantes banana:http://www.victoriacross.net
1189631332976976 Abantes onion:http://pbskids.org/sesame/index.html
1189631333347557 Abantes onion:http://www.baseball-reference.com/r/roberda07.shtml
1189631333149345 Abantes onion:http://www.eighttribes.org/modoc/
1189631333505118 Abantes onion:http://www.lsbu.ac.uk/news-php/news.php?newsid=86
1189631333169117 Abantes onion:http://www.mtv.com/news/articles/1508677/20050831/duff_hilary.jhtml?headlines=true
1189631333578939 Abantes cassis:http://haynese.winthrop.edu/india/medals/VC/1CSingh.html
1189631333156244 Abantes cassis:http://pbskids.org/sesame/index.html
1189631333031474 Abantes cassis:http://www.freebornhall.com/History/ResidenceHalls/
1189631333049811 Abantes cassis:http://www.guardian.co.uk/Columnists/Archive/0,5673,-25,0.html
1189631333361760 Abantes cassis:http://www.nielsenmedia.com/DMAs.html
1189631333554142 Abantes cassis:http://www.sirlinksalot.net/bertandernie.html
1189631333529093 Abantes cassis:http://www.sirlinksalot.net/bertandernie.html
1189631332877378 Abantes cassis:http://www.usatoday.com/sports/bbw/columnist/nightengale/2005-01-19-majors_x.htm
1189631333480657 Abantes cherry:http://sports.espn.go.com/mlb/players/profile?statsId=6305
1189631333405285 Abantes cherry:http://sports.espn.go.com/mlb/players/profile?statsId=6305
1189631333044738 Abantes cherry:http://www.eighttribes.org/modoc/
1189631333486673 Abantes cherry:http://www.freebornhall.com/History/ResidenceHalls/
1189631333500629 Abantes cherry:http://www.housing.berkeley.edu/housing/
1189631333589329 Abantes cherry:http://www.klamathtribes.org
1189631333450524 Abantes cherry:http://www.sirlinksalot.net/bertandernie.html
1189631333350277 Abantes cherry:http://www.usdoj.gov/usao/iln/osc/documents/ag_letter_december_30_2003.pdf
1189631332814749 Abantes cherry:http://www.usdoj.gov/usao/iln/osc/documents/ag_letter_december_30_2003.pdf
1189631333696564 Abantes yam:http://haynese.winthrop.edu/india/medals/VC/1CSingh.html
1189631333211111 Abantes yam:http://toughpigs.com/anthernie00.htm
1189631333408763 Abantes yam:http://www.sesameworkshop.org/sesamestreet/
1189631333646451 Abantes plum:http://toughpigs.com/anthernie00.htm
1189631333017569 Abantes plum:http://toughpigs.com/anthernie00.htm
1189631333062129 Abantes plum:http://www.sirlinksalot.net/bertandernie.html
1189631333449980 Abantes broccoli:http://www.fcc.gov/fcc-bin/tvq?call=KOCB
1189631333225383 Abantes broccoli:http://www.victoriacross.net
1189631333300379 Abarambo apple:http://sports.espn.go.com/mlb/players/profile?statsId=6305
1189631332840723 Abarambo apple:http://toughpigs.com/anthernie00.htm
1189631333045875 Abarambo apple:http://www.housing.berkeley.edu/housing/
1189631331863671 Abarambo apple:http://www.nielsenmedia.com/DMAs.html
1189631333199627 Abarambo apple:http://www.psa.ac.uk/awards2003/toynbee.htm
1189631332860025 Abarambo apple:http://www.psa.ac.uk/awards2003/toynbee.htm
1189631333167854 Abarambo apple:http://www.sesameworkshop.org/sesamestreet/
1189631332865498 Abarambo apple:http://www.sesameworkshop.org/sesamestreet/
1189631333361846 Abarambo apple:http://www.usatoday.com/sports/bbw/columnist/nightengale/2005-01-19-majors_x.htm
1189631333049487 Abarambo banana:http://haynese.winthrop.edu/india/medals/VC/1CSingh.html
1189631333474345 Abarambo banana:http://toughpigs.com/anthernie00.htm
1189631332754769 Abarambo banana:http://toughpigs.com/anthernie00.htm
1189631333138680 Abarambo banana:http://www.housing.berkeley.edu/housing/
1189631332660918 Abarambo banana:http://www.sirlinksalot.net/bertandernie.html
1189631333389776 Abarambo banana:http://www.victoriacross.net
1189631333579536 Abarambo onion:http://haynese.winthrop.edu/india/medals/VC/1CSingh.html
1189631332145125 Abarambo onion:http://sports.espn.go.com/mlb/players/profile?statsId=6305
1189631332933090 Abarambo onion:http://www.freebornhall.com/History/ResidenceHalls/
1189631333175393 Abarambo onion:http://www.guardian.co.uk/Columnists/Archive/0,5673,-25,0.html
1189631333177134 Abarambo onion:http://www.lsbu.ac.uk/news-php/news.php?newsid=86
1189631332776669 Abarambo onion:http://www.usatoday.com/sports/bbw/columnist/nightengale/2005-01-19-majors_x.htm
1189631332463173 Abarambo onion:http://www.usdoj.gov/usao/iln/osc/documents/ag_letter_december_30_2003.pdf
1189631333055414 Abarambo cassis:http://haynese.winthrop.edu/india/medals/VC/1CSingh.html
1189631332567593 Abarambo cassis:http://toughpigs.com/anthernie00.htm
1189631332548717 Abarambo cassis:http://www.guardian.co.uk/Columnists/Archive/0,5673,-25,0.html
1189631332230211 Abarambo cassis:http://www.psa.ac.uk/awards2003/toynbee.htm
1189631333541743 Abarambo cassis:http://www.usatoday.com/sports/bbw/columnist/nightengale/2005-01-19-majors_x.htm
1189631333382622 Abarambo cherry:http://pbskids.org/sesame/index.html
1189631333409030 Abarambo cherry:http://www.baseball-reference.com/r/roberda07.shtml
1189631331998971 Abarambo cherry:http://www.baseball-reference.com/r/roberda07.shtml
1189631332922366 Abarambo cherry:http://www.eighttribes.org/modoc/
1189631333667297 Abarambo cherry:http://www.fcc.gov/fcc-bin/tvq?call=KOCB
1189631332540031 Abarambo cherry:http://www.fcc.gov/fcc-bin/tvq?call=KOCB
1189631333615524 Abarambo cherry:http://www.housing.berkeley.edu/housing/
1189631332617084 Abarambo cherry:http://www.lsbu.ac.uk/news-php/news.php?newsid=86
1189631331968861 Abarambo cherry:http://www.lsbu.ac.uk/news-php/news.php?newsid=86
1189631331870788 Abarambo cherry:http://www.lsbu.ac.uk/news-php/news.php?newsid=86
1189631332842340 Abarambo cherry:http://www.mtv.com/news/articles/1508677/20050831/duff_hilary.jhtml?headlines=true
1189631332438657 Abarambo cherry:http://www.mtv.com/news/articles/1508677/20050831/duff_hilary.jhtml?headlines=true
1189631332227541 Abarambo cherry:http://www.psa.ac.uk/awards2003/toynbee.htm
1189631333459814 Abarambo cherry:http://www.usdoj.gov/usao/iln/osc/documents/ag_letter_december_30_2003.pdf
1189631333301899 Abarambo cherry:http://www.victoriacross.net
1189631333026152 Abarambo yam:http://toughpigs.com/anthernie00.htm
1189631333317550 Abarambo yam:http://www.baseball-reference.com/r/roberda07.shtml
1189631332186061 Abarambo yam:http://www.baseball-reference.com/r/roberda07.shtml
1189631332222633 Abarambo yam:http://www.fcc.gov/fcc-bin/tvq?call=KOCB
1189631332616469 Abarambo yam:http://www.guardian.co.uk/Columnists/Archive/0,5673,-25,0.html
1189631333686718 Abarambo yam:http://www.klamathtribes.org
1189631332871422 Abarambo yam:http://www.klamathtribes.org
1189631333616840 Abarambo yam:http://www.lsbu.ac.uk/news-php/news.php?newsid=86
1189631333602157 Abarambo yam:http://www.lsbu.ac.uk/news-php/news.php?newsid=86
1189631333183085 Abarambo yam:http://www.nielsenmedia.com/DMAs.html
1189631333413995 Abarambo yam:http://www.sesameworkshop.org/sesamestreet/
1189631332320721 Abarambo yam:http://www.sesameworkshop.org/sesamestreet/
1189631333164762 Abarambo yam:http://www.sirlinksalot.net/bertandernie.html
1189631333300493 Abarambo yam:http://www.usatoday.com/sports/bbw/columnist/nightengale/2005-01-19-majors_x.htm
1189631331836281 Abarambo yam:http://www.victoriacross.net
1189631332836152 Abarambo plum:http://haynese.winthrop.edu/india/medals/VC/1CSingh.html
1189631333233699 Abarambo plum:http://pbskids.org/sesame/index.html
1189631332424639 Abarambo plum:http://sports.espn.go.com/mlb/players/profile?statsId=6305
1189631333209916 Abarambo plum:http://toughpigs.com/anthernie00.htm
1189631332982680 Abarambo plum:http://www.baseball-reference.com/r/roberda07.shtml
1189631332701169 Abarambo plum:http://www.eighttribes.org/modoc/
1189631333510796 Abarambo plum:http://www.housing.berkeley.edu/housing/
1189631333354943 Abarambo plum:http://www.klamathtribes.org
1189631331861154 Abarambo plum:http://www.klamathtribes.org
1189631331948574 Abarambo broccoli:http://haynese.winthrop.edu/india/medals/VC/1CSingh.html
1189631333387254 Abarambo broccoli:http://pbskids.org/sesame/index.html
1189631332256737 Abarambo broccoli:http://www.eighttribes.org/modoc/
1189631332677347 Abarambo broccoli:http://www.guardian.co.uk/Columnists/Archive/0,5673,-25,0.html
1189631332950863 Abarambo broccoli:http://www.nielsenmedia.com/DMAs.html
1189631332442202 Abarambo broccoli:http://www.nielsenmedia.com/DMAs.html
1189631333623650 Abarambo broccoli:http://www.sesameworkshop.org/sesamestreet/
DESTROY SCANNER;
CREATE SCANNER ON 'test/Test2'[..??] WHERE ROW >= "Abama" LIMIT=2 REVS=1;
1189631333141746 Abama apple:http://haynese.winthrop.edu/india/medals/VC/1CSingh.html
1189631333575431 Abama apple:http://www.baseball-reference.com/r/roberda07.shtml
1189631333438063 Abama apple:http://www.usatoday.com/sports/bbw/columnist/nightengale/2005-01-19-majors_x.htm
1189631333192949 Abama banana:http://www.baseball-reference.com/r/roberda07.shtml
1189631333239488 Abama banana:http://www.fcc.gov/fcc-bin/tvq?call=KOCB
1189631333289762 Abama banana:http://www.psa.ac.uk/awards2003/toynbee.htm
1189631333632042 Abama onion:http://sports.espn.go.com/mlb/players/profile?statsId=6305
1189631332983672 Abama onion:http://www.guardian.co.uk/Columnists/Archive/0,5673,-25,0.html
1189631333634436 Abama onion:http://www.housing.berkeley.edu/housing/
1189631333105993 Abama onion:http://www.mtv.com/news/articles/1508677/20050831/duff_hilary.jhtml?headlines=true
1189631333331091 Abama onion:http://www.nielsenmedia.com/DMAs.html
1189631333416064 Abama onion:http://www.usatoday.com/sports/bbw/columnist/nightengale/2005-01-19-majors_x.htm
1189631333479027 Abama cassis:http://haynese.winthrop.edu/india/medals/VC/1CSingh.html
1189631333309782 Abama cassis:http://pbskids.org/sesame/index.html
1189631333225486 Abama cassis:http://sports.espn.go.com/mlb/players/profile?statsId=6305
1189631333314019 Abama cassis:http://www.eighttribes.org/modoc/
1189631333487606 Abama cassis:http://www.klamathtribes.org
1189631333053367 Abama cassis:http://www.usdoj.gov/usao/iln/osc/documents/ag_letter_december_30_2003.pdf
1189631333511132 Abama cherry:http://toughpigs.com/anthernie00.htm
1189631333180300 Abama cherry:http://www.guardian.co.uk/Columnists/Archive/0,5673,-25,0.html
1189631333626253 Abama cherry:http://www.klamathtribes.org
1189631333049254 Abama cherry:http://www.lsbu.ac.uk/news-php/news.php?newsid=86
1189631333362748 Abama cherry:http://www.sesameworkshop.org/sesamestreet/
1189631333682271 Abama yam:http://haynese.winthrop.edu/india/medals/VC/1CSingh.html
1189631333379272 Abama yam:http://pbskids.org/sesame/index.html
1189631333064725 Abama yam:http://toughpigs.com/anthernie00.htm
1189631333228657 Abama yam:http://www.klamathtribes.org
1189631333659574 Abama yam:http://www.mtv.com/news/articles/1508677/20050831/duff_hilary.jhtml?headlines=true
1189631333397374 Abama yam:http://www.psa.ac.uk/awards2003/toynbee.htm
1189631333075652 Abama yam:http://www.sesameworkshop.org/sesamestreet/
1189631333175254 Abama yam:http://www.sirlinksalot.net/bertandernie.html
1189631333674696 Abama plum:http://pbskids.org/sesame/index.html
1189631333353299 Abama plum:http://www.baseball-reference.com/r/roberda07.shtml
1189631333025117 Abama broccoli:http://www.sirlinksalot.net/bertandernie.html
1189631333633440 Abama broccoli:http://www.usdoj.gov/usao/iln/osc/documents/ag_letter_december_30_2003.pdf
1189631333707493 Abanic banana:http://haynese.winthrop.edu/india/medals/VC/1CSingh.html
1189631333605956 Abanic cassis:http://www.housing.berkeley.edu/housing/
1189631333670766 Abanic cherry:http://www.guardian.co.uk/Columnists/Archive/0,5673,-25,0.html
1189631333532516 Abanic cherry:http://www.housing.berkeley.edu/housing/
1189631333678356 Abanic yam:http://sports.espn.go.com/mlb/players/profile?statsId=6305
1189631333561403 Abanic yam:http://www.mtv.com/news/articles/1508677/20050831/duff_hilary.jhtml?headlines=true
DESTROY SCANNER;
